# These limits are used by MoveIt and augment/override the definitions in ur_description.
#
# While the robot does not inherently have any limits on joint accelerations (only on torques),
# MoveIt needs them for time parametrization. They were chosen conservatively to work in most use
# cases. For specific applications, higher values might lead to better execution performance.

joint_limits:
  shoulder_pan_joint:
    has_acceleration_limits: true
    max_acceleration: 5.0
  shoulder_lift_joint:
    has_acceleration_limits: true
    max_acceleration: 5.0
  elbow_joint:
    has_acceleration_limits: true
    max_acceleration: 5.0
  wrist_1_joint:
    has_acceleration_limits: true
    max_acceleration: 5.0
    # Adjusted to avoid crashing OCT/OCE equipment, min: -270, max: 0
    # has_position_limits: true
    # max_position: 0.0
    # min_position: -4.71
  wrist_2_joint:
    has_acceleration_limits: true
    max_acceleration: 5.0
    # Adjusted to avoid too much movement, min: -270, max: 90
    # has_position_limits: true
    # max_position: 1.57
    # min_position: -4.71
  wrist_3_joint:
    has_acceleration_limits: true
    max_acceleration: 5.0
